contributing_to_docs/term_glossary.adoc

```diff
@@ -34,109 +37,128 @@ An action consists of _project_, _verb_, and _resource_:
 * *Project* is the project containing the resource that is to be acted upon.
 * *Verb* is a get, list, create, or update operation.
 * *Resource* is the API endpoint being accessed. This is distinct from the
-referenced resource itself, which can be a pod, deployment config, build, etc.
+referenced resource itself, which can be a Pod, DeploymentConfig, Build, etc.
 
 ''''
 === apiserver
 
-Usage: apiserver(s) as appropriate
+Usage: apiserver(s) or API server(s) as appropriate
 
-A REST API endpoint for interacting with the system.  New deployments and
+A REST API endpoint for interacting with the system. New deployments and
 configurations can be created with this endpoint, and the state of the system
 can be interrogated through this endpoint as well.
 
 ''''
+=== app
+
+Usage: app(s)
+
+Acceptable when referring to a mobile or web xref:application[application].
+
+''''
+[[application]]
 === application
 
-Usage: _application(s)_ as appropriate; italicized
+Usage: application(s)
 
-Although the term _application_ is no longer an official noun in OpenShift,
+Although the term application is no longer an official noun in OpenShift,
 customers still create and host applications on OpenShift, and using the term
-within certain contexts is acceptable. For example, the term _application_ may
+within certain contexts is acceptable. For example, the term application may
```

Comment:
I'd s/may/might or s/may/can in all instances of "may" because it's not allowed by the style guide.